### PR TITLE
Update gaussian_naive_bayes.py

### PR DESCRIPTION
**Pull request by mvsg2**
Just adding in a final metric of accuracy to declare...

### Describe your change:
I have imported another module from sklearn.metrics library i.e. accuracy_score to display the final accuracy of the trained model out on the console. Also, just to make it a little propped up, I also imported the time library and stopped the code-execution at two points in the middle (this won't affect the final accuracy of the model in any way).

* [ ] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### Checklist:
* [ ] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [ ] This pull request is all my own work -- I have not plagiarized.
* [ ] I know that pull requests will not be merged if they fail the automated tests.
* [ ] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [ ] All new Python files are placed inside an existing directory.
* [ ] All filenames are in all lowercase characters with no spaces or dashes.
* [ ] All functions and variable names follow Python naming conventions.
* [ ] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [ ] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [ ] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [ ] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

File: machine_learning/gaussian_naive_bayes.py [hunk 1]
```diff
@@ -25,20 +27,26 @@ def main():
 
     # Gaussian Naive Bayes
     nb_model = GaussianNB()
-    nb_model.fit(x_train, y_train)
+    time.sleep(2.9)
```
Comment (cclauss) [inline machine_learning/gaussian_naive_bayes.py:30]: Why are we waiting for 3 seconds?!?  Comments needed to be added to explain such things.

**Review by cclauss — APPROVED** [review]
